pkgs/dart_services/Dockerfile

```diff
@@ -0,0 +1,12 @@
+ARG PROJECT_ID
+ARG FLUTTER_CHANNEL
+FROM gcr.io/$PROJECT_ID/flutter:$FLUTTER_CHANNEL
```

Comment:
For my own understanding, this is where we now get the sdk to use that powers the different flutter channels (stable, beta, main).

Reply:
Yeah when this image builds it will be based on the images built by the Cloud Builders defined in `.cloud_build/flutter`. 

I made a mistake here, I think we want to make sure we are using the stable channel of the Dart SDK to run dart_services. Right now this code uses the Dart SDK from the Flutter SDK, so the main channel would be more likely to break.

Reply:
I don't have huge concerns with re-using the same sdk that we compile and analyze snippets with, with the one that we run the backend server on. Both `stable` and `beta` should be fairly stable, and the compilation aspects of the backend have a higher chance to break than does something like the server runtime.

(separately, I think we could move the language experiments to the beta channel, and remove support for the `main` channel, w/o any real lose of functionality for dartpad)

Reply:
Just following up here - the build for the main channel is breaking so it will likely be out of date more often than not. I'm open to removing support for the main channel too. It should be easy enough to add back, but we might want to configure the `beta` channel to support experiment flags.